File: week2/Week2App/app/src/main/java/com/rumad/week2app/MainActivity.java
package com.rumad.week2app;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.widget.FrameLayout;

public class MainActivity extends AppCompatActivity {
    FrameLayout frameLayout;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        frameLayout = findViewById(R.id.frame_layout_holder);

        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();

        fragmentTransaction.replace(R.id.frame_layout_holder, new WeatherListFragment());
        fragmentTransaction.addToBackStack(null);

        fragmentTransaction.commit();
    }

}

File: Hemanth/week7/Week7App/app/src/main/java/com/rumad/week7app/MainActivity.kt
package com.rumad.week7app

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import androidx.core.R.id.async
import androidx.room.Room
import com.rumad.week7app.db.AppDatabase
import com.rumad.week7app.db.dao.TodoDao
import com.rumad.week7app.db.dbinstance
import com.rumad.week7app.db.entities.Todo
import java.util.*
import kotlin.concurrent.thread

class MainActivity : AppCompatActivity() {
    lateinit var todoDao: TodoDao

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        todoDao = dbinstance.getDatabase(this).todoDao()

        thread {
            todoDao.insertAll(Todo(null, "Sample Todo", "Sample Description", Date().toString()))

            val todos = todoDao.getAll()

            Log.d(javaClass.name, "Finished inserting ${todos}")
        }


    }
}
